vim-tmux-navigator.tmux

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+# This snippet can be sourced from your tmux config and will be used by
+# tpm (https://github.com/tmux-plugins/tpm).
+
+has_vim="ps -o state= -o comm= -t '#{pane_tty}' \
+    | grep -iEq '^[^TXZ ]+ +(\S+\/)?g?(view|n?vim?x?)(diff)?$'"
+
+tmux bind-key -n C-h  if-shell "$has_vim" "send-keys C-h"  "select-pane -L"
+tmux bind-key -n C-j  if-shell "$has_vim" "send-keys C-j"  "select-pane -D"
+tmux bind-key -n C-k  if-shell "$has_vim" "send-keys C-k"  "select-pane -U"
+tmux bind-key -n C-l  if-shell "$has_vim" "send-keys C-l"  "select-pane -R"
+tmux bind-key -n C-\\ if-shell "$has_vim" "send-keys C-\\" "select-pane -l"
```

Comment:
Let's remove this in favor of merging the TPM config in #123 after we get this update in.